Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/qrcode/VaccinationCertificateV1.kt

```diff
@@ -0,0 +1,42 @@
+package de.rki.coronawarnapp.vaccination.core.qrcode
+
+import com.google.gson.annotations.SerializedName
+import org.joda.time.LocalDate
+
+data class VaccinationCertificateV1(
+    @SerializedName("ver") val version: String,
+    @SerializedName("nam") val nameData: NameData,
+    @SerializedName("dob") val dateOfBirth: LocalDate,
+    @SerializedName("v") val vaccinationDatas: List<VaccinationData>,
+) {
+
+    data class NameData(
+        @SerializedName("fn") val familyName: String?,
+        @SerializedName("fnt") val familyNameStandardized: String,
+        @SerializedName("gn") val givenName: String?,
+        @SerializedName("gnt") val givenNameStandardized: String?,
+    )
+
+    data class VaccinationData(
```

Comment:
How this will scale for `n` certificates ? is not it better to save it a DB instead ?

Reply:
There is a lot of changes happening to the attributes, storing it as JSON makes it easier to add/remove attributes. Database changes will require migrations each time.

The worst case scenario seems to be:
*  2 parents? + a few children 4?
* 1-3 vaccinations per
* 1-2 proof certificates per

Still seems small enough to handle via JSON, and we don't need any complex queries on the data, and we have to load the data each time anyways to show on the home screen.

I'd prefer to go ahead with JSON for now, should the requirements change unexpectedly, we can just exchange the storage layer.